I fail to build it locally

```
…
mkdir -p Perl-Module/lib/
cat DISCLAIMER >Perl-Module/lib/TakTuk.pm
./taktuk -p TakTuk >>Perl-Module/lib/TakTuk.pm
/nix/store/pnc8xc1dh3cy4migfipfmzy3ad7snz99-bash-4.3-p42/bin/bash: ./taktuk: /usr/bin/perl: bad interpreter: No such file or directory
Makefile:1009: recipe for target 'Perl-Module/lib/TakTuk.pm' failed
make[1]: *** [Perl-Module/lib/TakTuk.pm] Error 126
make[1]: Leaving directory '/tmp/nix-build-taktuk-3.7.5.drv-0/taktuk-3.7.5'
Makefile:509: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
builder for ‘/nix/store/g5gbljdgj2vwvb23ihab8fy1xzghqxc7-taktuk-3.7.5.drv’ failed with exit code 2
error: build of ‘/nix/store/g5gbljdgj2vwvb23ihab8fy1xzghqxc7-taktuk-3.7.5.drv’ failed
```

Maybe you should add `perl` to the `buildInputs` and add something `substituteInPlace taktuk "/usr/bin/perl" "${perl}/bin/perl` in the `patchPhase` ?

--- a/pkgs/applications/networking/cluster/taktuk/default.nix
+++ b/pkgs/applications/networking/cluster/taktuk/default.nix
@@ -1,0 +1,34 @@
+{ stdenv, fetchurl, perl , openssh}:
+
+stdenv.mkDerivation rec {
+  version = "3.7.5";
+  name = "taktuk-${version}";
+
+  buildInputs = [ perl ];
+
+  src = fetchurl {
+    url = "https://gforge.inria.fr/frs/download.php/33412/${name}-3.7.5.tar.gz";
+    sha256 = "d96ef6c63d77f32339066f143ef7e0bc00041e10724254bf15787746ad1f1070";
+  };
+
+  preBuild = ''
+      substituteInPlace ./taktuk --replace "/usr/bin/perl" "${perl}/bin/perl"
+  '';
+
+  meta = {
+    description = "Efficient, large scale, parallel remote execution of commands";
+    longDescription = ''
+      TakTuk allows one to execute commands in parallel on a potentially large set
+      of remote nodes (using ssh to connect to each node). It is typically used
+      inside high performance computing clusters and grids. It uses an adaptive
+      algorithm to efficiently distribute the work and sets up an interconnection
+      network to transport commands and perform I/Os multiplexing. It doesn't
+      require any specific software on the nodes thanks to a self-propagation
+      algorithm.'';
+    homepage = http://taktuk.gforge.inria.fr/;
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = [ stdenv.lib.maintainers.bzizou ];
+    platforms = stdenv.lib.platforms.linux;
+  };
+}
+

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -5451,6 +5451,8 @@ let
 
   supercollider_scel = supercollider.override { useSCEL = true; };
 
+  taktuk = callPackage ../applications/networking/cluster/taktuk { };
+
   tcl = tcl-8_6;
   tcl-8_5 = callPackage ../development/interpreters/tcl/8.5.nix { };
   tcl-8_6 = callPackage ../development/interpreters/tcl/8.6.nix { };

